package-res/ccc/core/base/chart/chart.activeScene.js

```diff
@@ -57,18 +57,20 @@ pvc.BaseChart
      * @private
      */
     _activeSceneChange: function(ctx) {
-        var from = ctx.event.from, to = ctx.event.to;
+        this.useTextMeasureCache(function() {
+            var from = ctx.event.from, to = ctx.event.to;
 
-        // Change
-        if(from) from._clearActive();
-        if((this._activeScene = to)) to._setActive(true);
+            // Change
+            if(from) from._clearActive();
+            if((this._activeScene = to)) to._setActive(true);
 
-        // Render
-        // Unless from same panel (<=> same root scene).
-        if(from && (!to || to.root !== from.root))
-            from.panel().renderInteractive();
+            // Render
+            // Unless from same panel (<=> same root scene).
+            if(from && (!to || to.root !== from.root))
+                from.panel().renderInteractive();
 
-        if(to) to.panel().renderInteractive();
+            if(to) to.panel().renderInteractive();
+        }, ctx);
```

Comment:
Don't think you need to specify this last argument to `useTextMeasureCache`. You're not using `this` inside the passed function.